core/shared/src/main/scala/rapture/core/modes.scala

```diff
@@ -38,7 +38,7 @@ object Mode extends Mode_1 {
     msg = "No implicit mode was available for $"+"{Group} methods. " +
         "Please import a member of rapture.core.modes, e.g. modes.throwExceptions.")
 trait Mode[+Group <: MethodConstraint] { mode =>
-  type Wrap[+ _, _ <: Exception]
+  type Wrap[_, _ <: Exception]
```

Comment:
Was there a problem with covariance on this parameter in Scala 2.12? I can't remember the details, but I thought it was necessary...